e2e/cypress/integration/integrations/outgoing_webhook/disable_override_username_profile_spec.js

```diff
@@ -0,0 +1,643 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @outgoing_webhook
+/* eslint-disable max-lines */
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+
+import {enableUsernameAndIconOverrideInt, enableUsernameAndIconOverride} from '../incoming_webhook/helpers';
+
+describe('Outgoing webhook', () => {
+    let sysadmin;
+    let testTeam;
+    let testChannel;
+    let testUser;
+    let otherUser;
+    let siteName;
+
+    const triggerWord = 'text';
+    const triggerWordExpanded = 'text with some more text';
+    const callbackUrl = 'https://outgoing-webhook-01snptsk6qvz.runkit.sh/';
```

Comment:
Instead of using this external URL, please create a new endpoint at [webhook_server.js](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/webhook_serve.js) based from [the source](https://github.com/mattermost/mattermost-qa-automation/blob/master/server/release_test_webhook/index.js).  Once added, please add `cy.requireWebhookServer();` ad `before` hook so that it will throw a meaningful error when no local webhook server is running.

Reply:
sure, should be done now.